src/realm/error_codes.hpp

```diff
@@ -0,0 +1,43 @@
+/*************************************************************************
+ *
+ * Copyright 2021 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ **************************************************************************/
+
+#pragma once
+
+#include <realm/string_data.hpp>
+
+namespace realm {
+
+/*
+ * TODO As part of RCORE-720, we'll move all the exception types and error codes we want to expose in the
```

Comment:
Refer to GH issues instead of Jira issues

Reply:
RCORE-720 is an epic ticket, do those have GH issues?

Reply:
Ahh, no :-(. Let's leave it for now, it will be fixed "shortly" I assume.